### PR TITLE
Set active default to true ls-miners

### PR DESCRIPTION
**Pull request by Sriep**
A brief description of the changes in this PR:

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- 0chain:
- Other: ...

### CODE REVIEW

**Review by Kishan-Dhakan — COMMENTED** [review]
Whether the user uses `--active` or not, they will always get active miners. I suggest to change the flag usage corresponding to the default value change, make it something like `--all` that returns both active and inactive providers if passed.

**Comment by Sriep** [comment]
`ls-miners` or `ls-miners --active=ture` now returns active miners, and `ls-mienrs --active=false` returns inactive miners.

This is a change from the current situation where instead `ls-mines` defaults to inactive miners.

**Comment by Kishan-Dhakan** [comment]
That makes sense but boolean flags are generally used such that if they're used they're true and if not used, false. Anyway it's subjective so I'll leave it with you

File: cmd/minersc.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ var minerscMiners = &cobra.Command{
 		)
 
 		limit, offset := 20, 0
-		active := false
+		active := true
```
Comment (peterlimg) [inline cmd/minersc.go:71]: Please update default flag value in line 627 and 633 as well, otherwise users could get confused.

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**